src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -51,6 +51,10 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
             sb.Append(TelemetryConstants.CommaDelimiter);
             // Token type is to indicate 1 - bearer, 2 - pop, 3 - ssh-cert, 4 - external.
             sb.Append(eventInProgress.TokenTypeString);
+            sb.Append(TelemetryConstants.CommaDelimiter);
```

Comment:
Should this be wrapped in a null check for these values? Or is it okay to have the string with the empty sections? e.g. ",1,,"?

Who is parsing this data out? How are they using it?

Reply:
We parse this data and when a value is not available we expect the empty spaces. Data is parsed as a sequence.